tests/Donation/TransactionSubscriberTest.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Tests\AppBundle\Donation;
+
+use AppBundle\DataFixtures\ORM\LoadDonationData;
+use AppBundle\Repository\DonationRepository;
+use AppBundle\Repository\TransactionRepository;
+use Tests\AppBundle\Controller\ControllerTestTrait;
+use Tests\AppBundle\MysqlWebTestCase;
+
+/**
+ * @group functional
+ */
+class TransactionSubscriberTest extends MysqlWebTestCase
```

Comment:
You can extend `Tests\AppBundle\SqliteWebTestCase` if these tests do not require mysql-specific functions.
Functional test suite will be faster \o/